docs/_guide/rendering.md

```diff
@@ -84,3 +84,27 @@ class HelloWorldElement extends HTMLElement {
 ```
 
 Here, instead of declaring our template in HTML, we can do so in JS, and achieve exactly the same effect. We aren't using `@targets` in this example, as there is a more direct way to handle the data; re-calling `update()` will efficiently update only the parts that change.
+
+The same effect could be achieved using `@attr` via:
```

Comment:
Maybe it's worth switching these examples around? 

We generally structure the docs such that the examples with the most abstractions go toward the top, then the further down the page the more "low level" they become.

Reply:
Done in 57f15d4